src/BloomBrowserUI/bookEdit/js/origami.js

```diff
@@ -75,14 +77,16 @@ function layoutToggleClickHandler() {
 // Event handler to split the current box in half (vertically or horizontally)
 function splitClickHandler() {
     var myInner = $(this).closest('.split-pane-component-inner');
-    if ($(this).hasClass('splitter-top'))
+    if ($(this).hasClass('add-top'))
```

Comment:
"Splitter" is commonly used as the thing you drag to change the relative proportions of two things. That's not what this is. So that name is already taken, and re-using it was just a constant source of confusion.